.gitignore

```diff
@@ -176,4 +176,7 @@ typings/
 .fusebox/
 
 # DynamoDB Local files
-.dynamodb/
\ No newline at end of file
+.dynamodb/
+
+# package-lock.json
+package-lock.json
```

Comment:
Also, are we suppose to ignore the package-lock henceforth

Reply:
Yes